snuba/clickhouse/native.py

```diff
@@ -20,16 +23,39 @@
 from snuba.utils.metrics.wrapper import MetricsWrapper
 
 logger = logging.getLogger("snuba.clickhouse")
+trace_logger = logging.getLogger("clickhouse_driver.log")
+trace_logger.setLevel("INFO")
 
 Params = Optional[Union[Sequence[Any], Mapping[str, Any]]]
 
 metrics = MetricsWrapper(environment.metrics, "clickhouse.native")
 
 
+ClickhouseProfile = TypedDict(
```

Comment:
nit: why not a dataclass?

Reply:
Because it gets converted to a dictionary in `NativeReader` and it felt like a lot of unnecessary conversion.